### PR TITLE
BGL: fix orientation of make_icosahedron()

### LINKED ISSUE #2975: Vertex ordering in function make_icosahedron() is wrong
## Issue Details

The vertex ordering is cw instead of ccw such that all face normals are inverted, see picture below:

I inverted the ordering of one face and its normal is correct.

## Source Code

I output the mesh into an off file as follows

```
using Kernel = CGAL::Simple_cartesian<float>;
using Point_3 = Kernel::Point_3;
using Mesh = CGAL::Surface_mesh<Point_3>;
Mesh mesh;
CGAL::make_icosahedron<Mesh>(mesh, Point_3(0, 0, 0));
std::ofstream ofs("path/to/mesh.off");
ofs << mesh;
```

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 4.11.1
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Orient icosahedron CW to fit the orientation of other helpers.

## Release Management

* Affected package(s):BGL
* Issue(s) solved (if any): fix #2975

### CODE REVIEW

**Comment by maxGimeno** [comment]
I forgot to base this PR on 4.11 in Github at its creation, so Travis won't work and I don't feel like polluting the history by closing it and opening another, a PR, as I know it works. I just inverted some indices and tested the result with the Polyhedron demo and meshlab, and everything is fine.

File: BGL/include/CGAL/boost/graph/helpers.h [hunk 1]
```diff
@@ -737,7 +737,7 @@ bool is_degenerate_triangle_face(
 
 /**
  * \ingroup PkgBGLHelperFct
- * \brief Creates a triangulated regular prism
+ * \brief Creates a triangulated regular prism, cw oriented,
```
Comment (sloriot) [inline BGL/include/CGAL/boost/graph/helpers.h]: what is cw oriented? Don't we have a function called `is_outward_oriented()`?

Comment (maxGimeno) [reply BGL/include/CGAL/boost/graph/helpers.h]: cw is clockworck oriented.

Comment (sloriot) [reply BGL/include/CGAL/boost/graph/helpers.h]: If you meant clockwise, It does not mean anything in 3D space. Use outward oriented.

Comment (maxGimeno) [reply BGL/include/CGAL/boost/graph/helpers.h]: Yes I meant clockwise.

**Comment by lrineau** [comment]
Please tell when you rebase a PR, because Github does not send any notification.

**Comment by lrineau** [comment]
Successfully tested in [CGAL-4.12-Ic-218][*], and then merged (in both the release branch and `master`)

  [*]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-218.shtml

**Pull request merged**